pkg/image/apis/image/types.go

```diff
@@ -28,6 +28,15 @@ const (
 	// ExcludeImageSecretAnnotation indicates that a secret should not be returned by imagestream/secrets.
 	ExcludeImageSecretAnnotation = "openshift.io/image.excludeSecret"
 
+	// DockerImageLayersOrderAnnotation describes layers order in the docker image.
+	DockerImageLayersOrderAnnotation = "openshift.io/image.dockerLayersOrder"
+
+	// DockerImageLayersOrderAscending means that an image layers in the ascending order.
```

Comment:
indicates that image layers are sorted in the order of their addition (from oldest to latest)